etna/core/wagtail_hooks.py

```diff
@@ -9,3 +10,10 @@ def editor_css():
     return format_html(
         '<link rel="stylesheet" href="{}">', static("css/dist/etna-wagtail-editor.css")
     )
+
+
+@hooks.register("insert_global_admin_css")
+def global_admin_css():
+    if settings.FEATURE_PLATFORM_ENVIRONMENT_TYPE in ["development", "staging"]:
```

Comment:
Note: A similar style of feedback applies to this line, we could just check if it's production, otherwise show the colours.

These changes are non-blocking. From our original conversation, I thought there was going to be multiple different styling (dev + staging).